plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1248,7 +1247,7 @@ func PodToleratesNodeTaints(pod *v1.Pod, meta interface{}, nodeInfo *schedulerca
 	return false, []algorithm.PredicateFailureReason{ErrTaintsTolerationsNotMatch}, nil
 }
 
-// Determine if a pod is scheduled with best-effort QoS
+// Check if pod is BestEffort pod
```

Comment:
please follow go standard for func comments:

isPodBestEffort checks if pod is scheduled with best-effort QoS